nva-datamodel-java/src/main/java/no/unit/nva/model/PublicationStatus.java

```diff
@@ -9,6 +9,7 @@ public enum PublicationStatus {
 
     NEW("NEW"),
     DRAFT("DRAFT"),
+    PUBLISHED_METADATA("PUBLISHED_METADATA"),
```

Comment:
Vil ikke dette i praksis fjerne behovet for `PUBLISHED`?
Antar klienten fortsatt skal forholde seg til `PublishedFile`/`UnpublishedFile` for filer, så `status` vil uansett representere metadata slik jeg forstår det